packages/jest-mock/__typetests__/mock-functions.test.ts

```diff
@@ -492,3 +494,65 @@ expectError(
     (key: string, value: number) => {},
   ),
 );
+
+// replaceProperty + Replaced
```

Comment:
These tests look good. Thanks.

Could you add simple e2e test for `jest.replaceProperty`, like this one (to the same `jest.test.ts` file):

https://github.com/facebook/jest/blob/fd8f89719912996802fe90bdeab1d5a6d8e95d8e/packages/jest-types/__typetests__/jest.test.ts#L263

And e2e tests for `jest. Replaced` as well. Also somewhere here (): https://github.com/facebook/jest/blob/fd8f89719912996802fe90bdeab1d5a6d8e95d8e/packages/jest-types/__typetests__/jest.test.ts#L265-L274